src/invidious/routes/misc.cr

```diff
@@ -40,14 +40,24 @@ module Invidious::Routes::Misc
 
   def self.cross_instance_redirect(env)
     referer = get_referer(env)
+    redirect_url = "redirect.invidious.io"
 
     instance_list = Invidious::Jobs::InstanceListRefreshJob::INSTANCES["INSTANCES"]
     if instance_list.empty?
-      instance_url = "redirect.invidious.io"
+      instance_url = redirect_url
     else
       # Sample returns an array
       # Instances are packaged as {region, domain} in the instance list
-      instance_url = instance_list.sample(1)[0][1]
```

Comment:
Please move the two comments that were above this line to line to the new location of `.sample(1)[0][1]`!